<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res.qrc">
    <normaloff>:/icons/settings.ico</normaloff>:/icons/settings.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="indentation">
      <number>10</number>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>Settings</string>
      </property>
      <property name="text">
       <string notr="true">Settings</string>
      </property>
      <item>
       <property name="text">
        <string>Startup</string>
       </property>
       <property name="text">
        <string notr="true">Settings.Startup</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Connection</string>
       </property>
       <property name="text">
        <string notr="true">Settings.Connection</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>User</string>
      </property>
      <property name="text">
       <string notr="true">User</string>
      </property>
      <item>
       <property name="text">
        <string>Accounts</string>
       </property>
       <property name="text">
        <string notr="true">User.Accounts</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>Display</string>
      </property>
      <property name="text">
       <string notr="true">Display</string>
      </property>
      <item>
       <property name="text">
        <string>Themes</string>
       </property>
       <property name="text">
        <string notr="true">Display.Themes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Roster</string>
       </property>
       <property name="text">
        <string notr="true">Display.Roster</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>History</string>
       </property>
       <property name="text">
        <string notr="true">Display.History</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chat window</string>
       </property>
       <property name="text">
        <string notr="true">Display.ChatWindow</string>
       </property>
       <item>
        <property name="text">
         <string>Tabs</string>
        </property>
        <property name="text">
         <string notr="true">Display.ChatWindow.Tabs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Smilies</string>
        </property>
        <property name="text">
         <string notr="true">Display.ChatWindow.Smilies</string>
        </property>
       </item>
      </item>
     </item>
     <item>
      <property name="text">
       <string>Plugins</string>
      </property>
      <property name="text">
       <string notr="true">Plugins</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="settingsPage">
         <property name="settingPage" stdset="0">
          <string notr="true">Settings</string>
         </property>
        </widget>
        <widget class="QWidget" name="settingsStartupPage">
         <property name="settingPage" stdset="0">
          <string notr="true">Settings.Startup</string>
         </property>
        </widget>
        <widget class="QWidget" name="settingsConnectionPage">
         <property name="settingPage" stdset="0">
          <string notr="true">Settings.Connection</string>
         </property>
        </widget>
        <widget class="QWidget" name="userPage">
         <property name="settingPage" stdset="0">
          <string notr="true">User</string>
         </property>
        </widget>
        <widget class="QWidget" name="userAccountsPage">
         <property name="settingPage" stdset="0">
          <string notr="true">User.Accounts</string>
         </property>
        </widget>
        <widget class="QWidget" name="displayPage">
         <property name="settingPage" stdset="0">
          <string notr="true">Display</string>
         </property>
        </widget>
        <widget class="QWidget" name="displayThemesPage">
         <property name="settingPage" stdset="0">
          <string notr="true">Display.Themes</string>
         </property>
        </widget>
        <widget class="QWidget" name="displayRosterPage">
         <property name="settingPage" stdset="0">
          <string notr="true">Display.Roster</string>
         </property>
        </widget>
        <widget class="QWidget" name="displayHistoryPage">
         <property name="settingPage" stdset="0">
          <string notr="true">Display.History</string>
         </property>
        </widget>
        <widget class="QWidget" name="displayChatWindowPage">
         <property name="settingPage" stdset="0">
          <string notr="true">Display.ChatWindow</string>
         </property>
        </widget>
        <widget class="QWidget" name="displayChatWindowTabsPage">
         <property name="settingPage" stdset="0">
          <string notr="true">Display.ChatWindow.Tabs</string>
         </property>
        </widget>
        <widget class="QWidget" name="displayChatWindowSmiliesPage">
         <property name="settingPage" stdset="0">
          <string notr="true">Display.ChatWindow.Smilies</string>
         </property>
        </widget>
        <widget class="QWidget" name="pluginsPage">
         <property name="settingPage" stdset="0">
          <string notr="true">Plugins</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
